components/Gallery/Gallery.js

```diff
@@ -1,21 +1,20 @@
 import React from 'react'
-
-import withT from '../../lib/withT'
-
-import photoswipeStyle from './photoswipeStyle'
-
-import { Spinner } from '@project-r/styleguide'
-
+import { compose } from 'react-apollo'
 import PhotoSwipe from 'photoswipe'
 import PhotoSwipeUIDefault from 'photoswipe/dist/photoswipe-ui-default'
-
 import { imageSizeInfo, imageResizeUrl } from 'mdast-react-render/lib/utils'
+import { Spinner } from '@project-r/styleguide'
+
+import withT from '../../lib/withT'
+import { withTester } from '../Auth/checkRoles'
+import photoswipeStyle from './photoswipeStyle'
+import { ZINDEX_LOADINGBAR } from '../constants'
```

Comment:
why `ZINDEX_LOADINGBAR`? IMO dedicated const or `zIndex.overlay` from the [styleguide](https://github.com/orbiting/styleguide/blob/44aceafe67794d7bd112ecce6fe7547fabcbaca8/src/theme/zIndex.js#L5).